api_docs/outgoing-webhooks.md

```diff
@@ -1,56 +1,52 @@
 # Outgoing webhooks
 
 Outgoing webhooks allow you to build or set up Zulip integrations
-which are notified when certain types of messages are sent in
-Zulip. When one of those events is triggered, we'll send an HTTP POST
-payload to the webhook's configured URL.  Webhooks can be used to
-power a wide range of Zulip integrations.  For example, the
-[Zulip Botserver][zulip-botserver] is built on top of this API.
-
-Zulip supports outgoing webhooks both in a clean native Zulip format,
-as well as a format that's compatible with
-[Slack's outgoing webhook API][slack-outgoing-webhook], which can help
-with porting an existing Slack integration to work with Zulip.
-
-[zulip-botserver]: /api/deploying-bots#zulip-botserver
-[slack-outgoing-webhook]: https://api.slack.com/custom-integrations/outgoing-webhooks
-
-To register an outgoing webhook:
-
-* Log in to the Zulip server.
-* Navigate to *Personal settings (<i class="zulip-icon zulip-icon-gear"></i>)* -> *Bots* ->
-  *Add a new bot*.  Select *Outgoing webhook* for bot type, the URL
-  you'd like Zulip to post to as the **Endpoint URL**, the format you
-  want, and click on *Create bot*. to submit the form/
-* Your new bot user will appear in the *Active bots* panel, which you
-  can use to edit the bot's settings.
+which are notified when certain types of messages are sent in Zulip
+When one of those events is [triggered](#triggering), we'll send an
```

Comment:
Nit: `s/we'll/the zulip Server will/` -- generally "we" or "us" isn't language we use in docs to specify what the software will do.